src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/takemanagement/viewmodel/AudioPluginViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package org.wycliffeassociates.otter.jvm.app.ui.takemanagement.viewmodel
+
+import io.reactivex.Single
+import org.wycliffeassociates.otter.common.data.workbook.Take
+import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.common.domain.content.*
+import org.wycliffeassociates.otter.common.domain.plugins.LaunchPlugin
+import org.wycliffeassociates.otter.jvm.app.ui.addplugin.view.AddPluginView
+import org.wycliffeassociates.otter.jvm.app.ui.addplugin.viewmodel.AddPluginViewModel
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import org.wycliffeassociates.otter.jvm.app.ui.workbook.viewmodel.WorkbookViewModel
+import org.wycliffeassociates.otter.jvm.persistence.WaveFileCreator
+import tornadofx.*
+
+class AudioPluginViewModel: ViewModel() {
+    private val injector: Injector by inject()
+    private val pluginRepository = injector.pluginRepository
+
+    private val workbookViewModel: WorkbookViewModel by inject()
+
+    private val launchPlugin = LaunchPlugin(pluginRepository)
+    private val recordTake = RecordTake(WaveFileCreator(), launchPlugin)
+    private val editTake = EditTake(launchPlugin)
+
+    fun record(recordable: Recordable): Single<RecordTake.Result> = recordTake
```

Comment:
https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting

Reply:
Made the changes!